doc/plugin.txt

```diff
@@ -4,41 +4,49 @@ Synopsis
 
 Manage project plugins
 
-    add <pluginid>|<directory>|<giturl> [...] ..... add specified plugins
-                                                    pluginid will be matched in --searchpath / registry
-                                                    directory is a directory containing a plugin
-                                                    giturl is a git repo containing a plugin
-
-        [--searchpath <directory>] ................ when looking up plugins by ID, look in this directory and
-                                                    each of its subdirectories for the plugin before hitting the registry.
-                                                    Multiple search paths can be used by either specifying the flag multiple
-                                                    times, or by separating paths with a delimiter (: on 'nix, ; on Windows).
-
-        [--noregistry] ............................ don't search the registry for plugins
-
-        [--link] .................................. when installing from a local path, creates a symbolic link to the plugin
-                                                    instead of copying files. The extent to which files are linked vs copied
-                                                    varies by platform. Useful for plugin development.
-        [--save] .................................. save the information for specified plugin into config.xml
-        [--shrinkwrap] ............................ used together with --save, saves the installed version number to config.xml
-
-
-
-    remove <pluginid> [...] ....................... remove plugins with the given IDs
+    add <plugin-spec> [...] ............ Add specified plugins
+        --searchpath <directory> ....... When looking up plugins by ID, look in this directory and
+                                         each of its subdirectories before hitting the registry.
+                                         Multiple search paths can be specified.
+
+        --noregistry ................... Don't search the registry for plugins
+
+        --link ......................... When installing from a local path, creates a symbolic link 
+                                         instead of copying files. The extent to which files are linked
+                                         varies by platform. Useful for plugin development.
```

Comment:
That flag is only really useful for plugin development. It lets you debug your plugin in a project. For Android, that means you can actually open your code in Android Studio (otherwise, the project wouldn't include all of the cordova-android java code). I say leave it in.